### ISSUE TITLE
Last Sync'd Never Displays anything other than 'None' - Mac OS 10.12

### ISSUE THREAD

**Issue by jeromeza** [sieren/QSyncthingTray#197]
Last Sync'd Never Displays anything other than 'None' - Mac OS 10.12

OS: mac OS 10.12
QSyncthingTray: v0.5.7
Synching: v0.14.16
Shows: Last Sync'd > None

I assume this is meant to show the last sync'd files - however it doesn't. It just always shows 'None'.

**Comment by sieren** [comment]
I'll have a look whether this is an easy fix for a potential 0.5.8 release, else it'll go in here: https://github.com/sieren/QSyncthingTray/issues/198

**Comment by sieren** [comment]
I couldnt reproduce this, but I have a hunch this is because of the way QST is using the REST  API right now (occasionally a few files might slip through). This will all change in the new version as I'm rewriting the connection stack as well.

**Comment by HalfVoxel** [comment]
Probably not particularly important since you are changing a lot for the 0.6 release anyway, but for me the 'Last Synced' item is constantly showing an incorrect item. Not None, but a file that was changed a long time ago, definitely not the latest item.

**Comment by sieren** [comment]
Yeah, I've seen this a few times myself and it's based on incorrect use of the REST API in legacy QST.
This will all be fixed in the new version, which I'm working on whenever I have time (unfortunately not a lot right now).

This seems to big of a story to actually consider fixing in the current QST. I hope you can live with this for a little longer ;)
